update whats new page & colors (#1496)
grommet/hpe-design-system: aries-site/src/pages/whats-new.mdx

<!-- BEGIN: What's New Entries -->

## March 15-19, 2021

> ### Created Pagination page

Reviewer: ```suggestion
### Published Pagination page
```

- Added guidance and usage examples for when to use [pagination](https://design-system.hpe.design/components/pagination).

> 

Reviewer: ```suggestion
- Published [Figma designs for Pagination](https://www.figma.com/file/slRqNvhJub0QlgN46zSXoY/HPE-Pagination-Component?node-id=1%3A2)

```

> ### Created Spinner page

Reviewer: ```suggestion
### Published Spinner page
```

> - Spinner Page was added onto the site. You can find more about the spinner [here](https://design-system.hpe.design/components/spinner)

Reviewer: ```suggestion
- Spinner Page was added onto the site. Read more about [spinner guidance](https://design-system.hpe.design/components/spinner)
```

> ### Resolve X-Frame-Options

Reviewer: We can remove this section. This wouldn't affect consumers of Design System.

- Resolved one of the security issues that were found. Read more about the issue [here](https://github.com/grommet/hpe-design-system/issues/1466) 

## March 8-9, 2021

### Conducted a design-sprint to discuss Page-Layouts 

## March 1-5, 2021

> ### Replaced the light/dark mode button.

Reviewer: We can remove this section. It doesn't really affect consumers of the Design System.

The light/dark toggle was in the footer after some discussion we moved it next to the search icon in the header.

> ### Ran security scan on the design-system site.

Reviewer: ```suggestion
### Ran OWASP security scan on the design-system site
```

> An open source tool was used in order to find any security vulnerabilities present in the web application.

Reviewer: ```suggestion
- The OWASP Security Scan was run to identify any security vulnerabilities in the Design System site. This is required to ensure full compliance with HPE standards.
```
